### PR TITLE
Allow iOS build to look through all provision types

### PR DESCRIPTION
**Pull request by Mitko-Kerezov**
Implements http://teampulse.telerik.com/view#item/290757
Merge after https://github.com/telerik/mobile-cli-lib/pull/339

### CODE REVIEW

**Comment by teobugslayer** [comment]
:+1:

**Comment by Fatme** [comment]
If development provision is found and the command is executed without `--download` option, we should notify the user that we will not generate QR code and will download the `.ipa` instead.

File: lib/services/build.ts [hunk 1]
```diff
@@ -420,7 +421,7 @@ export class BuildService implements Project.IBuildService {
 
 			this.$loginManager.ensureLoggedIn().wait();
 
-			if(this.$mobileHelper.isWP8Platform(platform) && this.$project.projectData.WPSdk && this.$project.projectData.WPSdk === "8.0" && helpers.versionCompare(this.$project.projectData.FrameworkVersion,"3.7.0") >= 0) {
+			if(this.$project.projectData.Framework === this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.Cordova && this.$mobileHelper.isWP8Platform(platform) && this.$project.projectData.WPSdk && this.$project.projectData.WPSdk === "8.0" && semver.gte(this.$project.projectData.FrameworkVersion,"3.7.0")) {
```
Comment (Fatme) [inline lib/services/build.ts]: This is a little bit strange to be here. Consider moving it to `cordova-project`.

**Comment by Mitko-Kerezov** [comment]
Ping @Fatme, comments addressed

**Comment by Fatme** [comment]
cool! :+1:

**Comment by Mitko-Kerezov** [comment]
run ci

**Comment by Mitko-Kerezov** [comment]
run ci

**Pull request merged**